src/gsy_e/gsy_e_core/util.py

```diff
@@ -299,16 +311,18 @@ def wrapped(*args, **kwargs):
 
 
 def recursive_retry(functor, retry_count, max_retries, *args, **kwargs):
+    """Recursive retry."""
```

Comment:
`Recursive function that retries the execution of "functor" function if an exception is raised from it.`